regolith/helpers/l_abstracthelper.py

```diff
@@ -77,29 +72,17 @@ def construct_global_ctx(self):
 
     def sout(self):
         rc = self.rc
-        collection = self.gtx["presentations"]
-
-        if rc.year is None:
-            rc.year = ''
-        if rc.location is None:
-            rc.location = ''
-        if rc.title is None:
-            rc.title = ''
+        presentations = self.gtx["presentations"]
 
-        for key in collection:
-            if rc.author in key[1]['authors']:
-                if "location" in key[1] and rc.location.lower() in key[1]["location"].lower():
-                    if rc.title.lower() in key[1]["title"].lower():
-                        if "end_year" in key[1] and "begin_year" in key[1]:
-                            if rc.year in str(key[1]["end_year"]) or rc.year in str(key[1]["begin_year"]):
-                                print("---------------------------------------")
-                                if "title" in key[1]:
-                                    print("Title: " + key[1]["title"]+"\n")
-                                a = ""
-                                for i in range(0, len(key[1]["authors"])):
-                                    auth=key[1]["authors"][i]
-                                    auth=get_person_contact(auth, self.gtx["people"], self.gtx["contacts"])
-                                    a = a + auth + ", "
-                                print(a)
-                                print("\nAbstract: " + key[1]["abstract"])
+        for presentation in presentations:
+            if rc.title.casefold() not in presentation.get('title').casefold():
+                continue
+            print("---------------------------------------")
+            print(f"Title: {presentation.get('title')}\n")
+            author_list=[author
+                         if not get_person_contact(author, self.gtx["people"], self.gtx["contacts"])
+                         else get_person_contact(author, self.gtx["people"], self.gtx["contacts"]).get('name')
+                         for author in presentation.get('authors')]
+            print(", ".join(author_list))
+            print(f"\nAbstract: {presentation.get('abstract')}")
```

Comment:
always finish code files with a line-feed (i.e., put cursor there and hit enter)